### DISCUSSION TITLE
HTTP 500 / Server Error on topmost repo view for logged in users

### DISCUSSION THREAD

**Discussion by breedx2** [community/community#83985] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

We have a repo for a community website here: https://github.com/dorkbotpdx/dorkbotpdx.org
It looks like users who have authenticated to github.com UI will receive an HTTP 500 (server error), while anonymous users will not. In other words, incognito mode works fine, but being logged in does not. Clearing cookies does not help.

Sub-pages like [pulls](https://github.com/dorkbotpdx/dorkbotpdx.org/pulls) and [commits](https://github.com/dorkbotpdx/dorkbotpdx.org/commits) work fine. Even the [file browser for project dirs](https://github.com/dorkbotpdx/dorkbotpdx.org/blob/main/jtag_2015) seems to work fine. The [/tree/main](https://github.com/dorkbotpdx/dorkbotpdx.org/tree/main) subpath does not.

This has been confirmed across firefox and chrome on both macos and linux, both with users in the org and users outside the org.

Does anybody NOT see a 500 for that repo?

I did search for related an found #76872 which seems similar, but different browsers with different extensions all experience this problem.

**Comment by breedx2** [comment]
[Forks of this repo](https://github.com/justinzero/dorkbotpdx.org) also appear to have this same issue, strongly suggesting something with the navigation code.

**Comment by breedx2** [comment]
Welp, as if by magic, the problem has worked itself out through no action on our part. Presumably a bug was fixed. Thanks.

**Reply by breedx2** [reply]
I spoke too soon -- it's back to being broken. Maybe we got lucky one some random build back in the past?

**Comment by look** [comment]
Thanks for reporting this. I am looking into which team to direct the problem to, but in the meanwhile I looked into it. I think the problem is that the repository has a `code_of_conduct` that is a directory, which is exposing a bug in our code of conduct detection code.

**Reply by look** [reply]
Actually, it's because it's a symlink to a directory.

**Reply by breedx2** [reply]
Thanks @look for looking into it. Please let us know if there's anything we can do to help with the investigation or fix.

**Reply by look** [reply, marked as answer]
I think I've figured out the problem. If you want to work around it until I can fix the issue, making the `code_of_conduct` in your root directory not a symlink to a directory will help. Alternatively, you could add a `.github/code_of_conduct` file - that should be picked up first and avoid the bug.

**Reply by breedx2** [reply]
Yeah, adding `.github/CODE_OF_CONDUCT` does seem to have worked around it. Thanks.

**Comment by jboone** [comment]
Late to the party, but doesn’t seem to matter if I’m logged in to GitHub or not. iPadOS + Firefox + authenticated == 500 error. iPadOS + Chrome + unauthenticated == 500 error. iPadOS + Safari + unauthenticated == 500 error. Struck out in all cases. Assuming these browsers are isolated (as they should be), this seems conclusive that being logged in or not isn’t a factor.

**Reply by breedx2** [reply]
Interesting! Thanks for the update. This is new behavior for me as well -- incognito tab used to render just fine, however now it also just returns 500, as well as the logged-in tab returning 500. Ouch.

**Comment by jboone** [comment]
Just checked behavior on Arch Linux: Firefox + authenticated == 500 error. Chromium + unauthenticated == 500 error. I have not yet seen this page load, in all the combinations I've cited here and in my last comment.
